lib/fastlane/plugin/semaphore/actions/setup_semaphore_action.rb

```diff
@@ -99,7 +105,7 @@ def self.is_supported?(platform)
         # See: https://docs.fastlane.tools/advanced/#control-configuration-by-lane-and-by-platform
         #
         # [:ios, :mac, :android].include?(platform)
-        [:ios, :android].include?(platform)
+        [:ios].include?(platform)
```

Comment:
AFAIK this would revert #3 and means "this plugin doesn't support Android"?

Reply:
Should not fail on android, I'll put that back.